micro-frontends/src/styles/carbon-theme.scss

```diff
@@ -133,4 +131,14 @@ $css--body: false;
 .bx--modal-close__icon {
   height: 25px;
   width: 25px;
-}
\ No newline at end of file
+}
+.bx--inline-notification{
+  position: fixed;
+  top: 80px;
+  left: 40%;
+  z-index: 1001;
```

Comment:
is it possible to have a z-index lower value? if it's parent has z-index value lower them have it as same or +1, depending on parent base